lib/tether/ops_linux.go

```diff
@@ -698,6 +731,13 @@ func ApplyEndpoint(nl Netlink, t *BaseOperations, endpoint *NetworkEndpoint) err
 		}
 	}
 
+	// notify our peers of the assigned address
+	err = gratuitous(t, link, endpoint)
```

Comment:
Should we put this in the deffered function on line 622? Due to the return on line 677, I'm curious to know if broadcast would occur in the following case (or if it's not needed here):

1) User deploys a VCH with dchp issuing it an address or x.x.x.1.
2) User upgrades & reconfigures or otherwise deploys a vch with **static** ip x.x.x.1, where this function exists on L677.

Will bridged containers will be unaware of the mac address change?

Reply:
I've moved the call to immediately after the endpoint is updated with the new address so that we make a good best effort to update ARP tables even if subsequent updates such as routing tables fails.

We do not need to care about the early return at 677 - that's a scenario where we are not assigning an IP address because the link is not managed by tether.

Reply:
Thanks for clarification!